io.osdn.simplesok.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Simple Sokoban
+GenericName=Simple Sokoban
+Comment=Simple Sokoban Game
+Icon=io.osdn.simplesok.svg
```

```suggestion
Icon=io.osdn.simplesok
```

icon name, not file name.